pro/src/app/AppRouter/AppRouter.tsx

```diff
@@ -13,11 +15,19 @@ const sentryCreateBrowserRouter =
 
 const AppRouter = (): JSX.Element => {
   const activeFeatures = useSelector(selectActiveFeatures)
+  const isNewInterfaceActive = useIsNewInterfaceActive()
 
   const activeRoutes = routes.filter(
     (route) => !route.featureName || activeFeatures.includes(route.featureName)
   )
 
+  useEffect(() => {
+    document.documentElement.setAttribute(
+      'data-theme',
+      isNewInterfaceActive ? 'new' : 'old'
```

Comment:
détail: je me méfie des noms "new" et "old" qui tendent à devenir obsolète genre "really-nexw-this-time" "very-old", !
ptet "pink" et "blue" ?

Reply:
+1